touchforms/formplayer/api.py

```diff
@@ -41,16 +41,22 @@ def __init__(self, username, password):
     def to_dict(self):
         return {'type': self.type, 'key': self.key, 'username': self.username}
 
-            
-class TouchformsException(Exception):
-    pass
 
-class InvalidSessionIdException(TouchformsException):
+class TouchformsError(ValueError):
```

Comment:
Inherits from ValueError so that anything that's currently catching ValueError still works